src/router_core/transfer.c

```diff
@@ -851,6 +977,144 @@ static void qdr_delete_delivery_CT(qdr_core_t *core, qdr_action_t *action, bool
         qdr_delete_delivery_internal_CT(core, action->args.delivery.delivery);
 }
 
+static bool qdr_delivery_is_all_peers_unlinked_CT(qdr_delivery_t *dlv)
```

Comment:
@ted-ross this function (qdr_delivery_is_all_peers_unlinked_CT) needs a little bit of your extra attention. Please take a look and let me know if you have questions.